ceph_deploy/install.py

```diff
@@ -37,7 +37,10 @@ def install(args):
 
     for hostname in args.host:
         LOG.debug('Detecting platform for host %s ...', hostname)
-        distro = hosts.get(hostname, username=args.username)
+        distro = hosts.get(
+            hostname,
+            username=args.username,
+            use_rhceph=bool(getattr(args, 'use_rhceph', False)))
```

Comment:
you are right -- I did not add a CLI switch.  probably should.  The expectation was to have that flag set in cephdeploy.conf by ice-setup, but there should be a switch for it as well.

The annoying thing if it is *not* in cephdeploy.conf is that you also need to do know if we are dealing with RH packages when doing purge and uninstall. I doubt someone wants to pass --use_rhceph if they do ceph-deploy uninstall...  We could just right the flag to the .conf file if we ever see it though, making it persistent.